monitoring/dashboards/viya/sas-arke-dashboard.json

```diff
@@ -0,0 +1,503 @@
+{
+    "annotations": {
+      "list": [
+        {
+          "builtIn": 1,
+          "datasource": {
+            "type": "grafana",
+            "uid": "-- Grafana --"
+          },
+          "enable": true,
+          "hide": true,
+          "iconColor": "rgba(0, 211, 255, 1)",
+          "name": "Annotations & Alerts",
+          "type": "dashboard"
+        }
+      ]
+    },
+    "description": "",
+    "editable": true,
+    "fiscalYearStartMonth": 0,
+    "graphTooltip": 0,
+    "id": 58,
+    "links": [],
+    "liveNow": false,
+    "panels": [
+      {
+        "datasource": {
+          "type": "prometheus",
+          "uid": "PBFA97CFB590B2093"
```

Comment:
Don't we have to change these to some generic datasource reference rather than this specific one?  a quick look at a few of our existing dashboards seems to show references like `"datasource": "Prometheus"`. Without making this change, I believe you will get error messages like: "
```
Templating Failed to upgrade legacy queries 
Datasource PBFA97CFB590B2093 was not found
```

Reply:
Yep, you are right.  I think this was a byproduct of how the dashboard was originally saved.
I have updated those references to be generic **$datasource** like our other dashboards.
It has been updated in the latest push.